docs/Documentation/Systems/Vermilion/modules.md

```diff
@@ -55,7 +55,8 @@ The recommended environment is enabled by running the source command:
 source /nopt/nrel/apps/210929a/myenv.2110041605
 ```
 
-**NOTE:  You may want to add this line to your .bashrc file so modules are available at login.**
+!!! note
```

Comment:
I'd suggest moving this earlier, even calling it a TL;DR

!!! note "TL;DR"
    Environments are not loaded by default for users, you must source your own.
    The recommended environment is enabled by running the source command:
    ```source /nopt/nrel/apps/210929a/myenv.2110041605```
    You may want to add this line to your .bashrc file so modules are available at login.